pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>org.liquibase.ext</groupId>
     <artifactId>liquibase-neo4j</artifactId>
-    <version>4.27.0-SNAPSHOT</version>
+    <version>4.26.0.1-SNAPSHOT</version>
```

Comment:
4.26.0.1 is already published, maybe keep 4.27.0-SNAPSHOT?

Reply:
I just deleted it to re-release same version, no need to release a different one @fbiville